cmd/ceremony/cert.go

```diff
@@ -61,13 +61,7 @@ type certProfile struct {
 	// policies extension. These should be formatted in the standard OID
 	// string format (i.e. "1.2.3"). This should only be used for policies
 	// which don't need any policyQualifiers.
```

Comment:
Need to update the comment here, which still says "This should only be used for policies which don't need any policyQualifiers."